src/MurmurClient.ts

```diff
@@ -169,12 +169,12 @@ export class MurmurClient {
         console.log(
           `Transaction included at blockHash ${result.status.asInBlock}`
         );
+        callback(result);
```

Comment:
why do we want the callback before the transaction is finalized?
Should we have two callbacks?

Reply:
We could have two callbacks, but the reason I moved it is UX. Waiting for it to be finalized just takes longer and being in a block is good enough for the demo imo.

Reply:
ok fair enough, let's leave it there for now